### PR TITLE
LPS-121567

### PR DESCRIPTION
**Pull request by petershin**

### CODE REVIEW

**Comment by petershin** [comment]
ci:test:bundle

**Comment by petershin** [comment]
ci:forward

File: modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java [hunk 1]
```diff
@@ -1028,7 +1028,8 @@ else if (devDependencies.containsKey("liferay-npm-scripts")) {
 			return false;
 		}
 
-		VersionNumber versionNumber = VersionNumber.parse(dependencyName);
+		VersionNumber versionNumber = VersionNumber.parse(
+			(String)devDependencies.get(dependencyName));
```
Comment (wincent) [inline modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java]: I feel so ashamed. Thanks for fixing!

ashamed

**Comment by wincent** [comment]
Going to try `ci:test:relevant` again — no unique failures, and I took the bundle for a test drive earlier and it seems fine.

**Comment by wincent** [comment]
ci:test:relevant

**Comment by petershin** [comment]
ci:forward

**Comment by wincent** [comment]
I see merge conflicts on the upstream PR, so I'm going to rebase this and resend to you.

**Comment by wincent** [comment]
Ok, rebased it and am running tests on it over here: https://github.com/liferay-frontend/liferay-portal/pull/409

🤞

**Pull request closed without merging**